src/pages/home/HomePage.js

```diff
@@ -205,8 +206,11 @@ class App extends React.Component {
     }
 
     render() {
-        const hamburgerStyle = this.state.isHamburgerEnabled && this.props.isSidebarShown
-            ? styles.hamburgerOpenAbsolute : styles.hamburgerOpen;
+        let hamburgerStyle = hamburgerInputPadding();
+        if (!hamburgerStyle) {
```

Comment:
Rather than do a conditional check here we should make the styles below return an empty object and just reference them normally. One way to do that would be to import them into the `styles` object. Then we do not need to perform these checks here and can leave everything as it was.

Please let me know if that makes sense.